### DISCUSSION TITLE
downstream CI for GraphMakie?

### DISCUSSION THREAD

**Discussion by anandijain** [JuliaPlots/Makie.jl#4215] [Ideas]
would a PR adding downstream CI for GraphMakie like https://github.com/SciML/ModelingToolkit.jl/blob/master/.github/workflows/Downstream.yml be accepted?

I ask because GraphMakie tests don't seem to pass with 0.17, maybe it'd be nice to know for a subset of dependents when things are breaking for that package. AoG may be good too

**Comment by asinghvi17** [comment]
I think it should be, maybe if it can be triggered manually somehow (maybe by a commit comment or PR comment)?  That way, we don't waste time if Makie's CI doesn't pass, and it's easier to tell if PRs have failed internal CI or the new downstream CI.

**Comment by ffreyer** [comment]
I think it would be interesting to have this for releases. Always running this would be too much though
